dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpConfigConstants.java

```diff
@@ -437,6 +437,16 @@ private HttpConfigConstants() {
 
     public static final String WILDCARD_CHAR = "*";
 
+    public static final String PROPNAME_HEADERS_FEATURE = "headersInternal";
+
```

Comment:
I'm not a huge fan of having the work `feature` in this naming convention. Can you work to come up with a better name here?

Reply:
Changed to `PROPNAME_RESPONSE_HEADERS`